variables.tf

```
@@ -63,6 +63,12 @@ variable "runtime" {
   description = "The function runtime to use"
 }
 
+variable "reserved_concurrency" {
+  type        = number
+  default     = null
+  description = "An optional reserved concurrent executions"
```

Could it be updated to this?

```suggestion
  description = "The amount of reserved concurrent executions for this lambda function"
```

I just copied this from the [resource argument docs](https://www.terraform.io/docs/providers/aws/r/lambda_function.html#reserved_concurrent_executions), it reads a bit better IMO.

---

thanks for suggestion and it is changed